src/util/resize.ts

```diff
@@ -0,0 +1,10 @@
+import {NgZone} from '@angular/core';
+import {fromEvent, Observable} from 'rxjs';
+import {takeUntil, throttleTime} from 'rxjs/operators';
+
+
+export function ngbWindowResize(zone: NgZone, callback: () => void, unsubscribe$: Observable<any>) {
+  zone.runOutsideAngular(() => {
+    fromEvent(window, 'resize').pipe(takeUntil(unsubscribe$), throttleTime(10)).subscribe(() => callback());
+  });
+}
```

Comment:
1) can we take advantage of the passive subscribers here, detecting browser support, and doing subscription based on that detection?
2) probably 'orientationchange' should be also involved into that positioning strategy for overlays:
e.g. (just sample, not the best code):
```diff
diff --git a/src/util/resize.ts b/src/util/resize.ts
index 59fd14b..d58e352 100644
--- a/src/util/resize.ts
+++ b/src/util/resize.ts
@@ -1,10 +1,33 @@
 import {NgZone} from '@angular/core';
-import {fromEvent, Observable} from 'rxjs';
+import {merge, fromEvent, Observable} from 'rxjs';
 import {takeUntil, throttleTime} from 'rxjs/operators';
 
+const passiveSupported: boolean = (() => {
+  let passiveSupported = false;
+  try {
+    const options = {
+      get passive() {
+        passiveSupported = true;
+        return true;
+      }
+    };
+    const listener = () => {};
+    window.addEventListener("test", listener, options);
+    window.removeEventListener("test", listener);
+  } catch (err) {
+    passiveSupported = false;
+  }
+  return passiveSupported;
+})();
 
 export function ngbWindowResize(zone: NgZone, callback: () => void, unsubscribe$: Observable<any>) {
   zone.runOutsideAngular(() => {
-    fromEvent(window, 'resize').pipe(takeUntil(unsubscribe$), throttleTime(10)).subscribe(() => callback());
+    const resizeObservable =
+        passiveSupported ? fromEvent(window, 'resize', {passive: true}) : fromEvent(window, 'resize');
+    const orientationChangeObservable = passiveSupported ? fromEvent(window, 'orientationchange', {passive: true}) :
+                                                           fromEvent(window, 'orientationchange');
+    merge(resizeObservable, orientationChangeObservable)
+        .pipe(takeUntil(unsubscribe$), throttleTime(10))
+        .subscribe(() => callback());
   });
 }
```